crypto/s2n_hash.c

```diff
@@ -89,17 +104,33 @@ int s2n_hash_digest_size(s2n_hash_algorithm alg, uint8_t *out)
 int s2n_hash_block_size(s2n_hash_algorithm alg, uint64_t *block_size)
 {
     POSIX_ENSURE(S2N_MEM_IS_WRITABLE_CHECK(block_size, sizeof(*block_size)), S2N_ERR_PRECONDITION_VIOLATION);
-    switch(alg) {
```

Comment:
These might be worth a clang-format off.

Reply:
I think the change is fine. Both are readable.